terra2/assetlist.json

```diff
@@ -586,6 +586,60 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/wsteth.svg"
       }
+    },
+    {
+      "description": "SEUL DAO",
+      "type_asset": "cw20",
+      "address": "terra13s5pxw5j2p4ssvzwvxd8l7h30vke8vjgtng75vqgv6p9vddfk3hskfka0l",
+      "denom_units": [
+        {
+          "denom": "cw20:terra13s5pxw5j2p4ssvzwvxd8l7h30vke8vjgtng75vqgv6p9vddfk3hskfka0l",
+          "exponent": 0
+        },
+        {
+          "denom": "SEUL",
```

Comment:
denoms are case sensitive. Can you please either lowercase this to denom to 'seul' so that the 'display' property is pointing ot it (or uppercase the display field to 'SEUL').

Reply:
applied for both tokens